doc/api/policy.md

```diff
@@ -196,4 +196,91 @@ module.exports = function fn(...args) {
 };
 ```
 
+### Scopes
+
+To set configuration for many resources at once you can use the `"scopes"`
+field of a manifest. `"scopes"` works by matching resources by their segments.
+If a dependency is not found, the resource may include `"cascade": true` which
+will delegate to a scope. Scopes may also use `"cascade": true`. The scope for
```

Comment:
These two sentences are a bit confusing and awkward together

Reply:
how about combining them:

> If a scope or resource includes `"cascade": true` unknown specifiers will
> be searched for in their containing scope.

Reply:
rephrased